data_export.py

```diff
@@ -40,6 +49,18 @@
             if 'ap' in result:
                 result['recall/ap'] = result['ap']
                 del result['ap']
+            if args.sensors:
+                if 'wspq' in result:
+                    result['wspq'] = result['wspq']
```

Comment:
This smells  a bit fishy. Do you intentionally want to make it explicit?

Reply:
Ah, I see now.  The "for" loop is for sanitizing the results dictionary for export.  Thanks!